src/Indice.Features.Messages.Core/Services/ContactService.cs

```diff
@@ -71,6 +60,50 @@ public async Task AddToDistributionList(Guid id, CreateDistributionListContactRe
         await DbContext.SaveChangesAsync();
     }
 
+    /// <inheritdoc />
+    public async Task BulkAddToDistributionList(Guid id, IEnumerable<CreateDistributionListContactRequest> requests) {
```

Comment:
We need here some kind of reporting about the number of affected contacts that where associated with the distribution list. Should consider a ContactImportResult class that encapsulates Success as well as counts and Errors if any.